editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -2378,8 +2378,9 @@ void Node3DEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 
 void Node3DEditorViewport::_nav_pan(Ref<InputEventWithModifiers> p_event, const Vector2 &p_relative) {
 	const NavigationScheme nav_scheme = (NavigationScheme)EDITOR_GET("editors/3d/navigation/navigation_scheme").operator int();
+	const real_t translation_sensitivity = EDITOR_GET("editors/3d/navigation_feel/translation_sensitivity");
 
-	real_t pan_speed = 1 / 150.0;
+	real_t pan_speed = 4 / 150.0 * translation_sensitivity;
```

Comment:
Changed 1 to 4 since default `translation_sensitivity` is 0.25

Reply:
Apologies in advance for the belated review - this PR should be mostly ready to merge, but I wanted to flag this. This seems to be fairly arbitrary, if I understand correctly `0.25` was chosen as the default simply because that's what `orbit_sensitivity` uses?

`orbit_sensitivity` seems to be in "degrees per pixel", so it's actually a meaningful unit.

Here `translation_sensitivity` seems to be in 1/37.5-th pixels per second (I assume), which seems pretty arbitrary.

Maybe we can make this setting actually be the full pixels per second value? Or maybe with a factor 100 because editing small decimals precisely is tricky.

So:
```suggestion
	real_t pan_speed = translation_sensitivity / 100.0;
```

And change the default value to `0.667`?

WDYT @Calinou?

Reply:
> Maybe we can make this setting actually be the full pixels per second value?

There is no time unit when panning, so it can't be "pixels per second" but would be something like "units per pixel" instead.

Reply:
I updated the PR to change the default translation sensitivity to `1.0`, and thus make the formula `translation_sensitivity / 150.0`.